compiler/modules/front.syntax/parseStmt.bal

```diff
@@ -221,18 +220,19 @@ function finishCallStmt(Tokenizer tok, CallStmt expr, Position startPos) returns
 function finishCheckingCallStmt(Tokenizer tok, CheckingKeyword checkingKeyword, Position startPos) returns CallStmt|err:Syntax {
```

Comment:
What is startPos the position of here? This needs a comment.

Reply:
`startPos` is the position of the `checkingKeyword` (parameter)